### DISCUSSION TITLE
Mobile.csproj created by template contains typo

### DISCUSSION THREAD

**Discussion by DrHardReset** [unoplatform/uno#15500] [Q&A] [answered]
Hey,

I just created a new UNO project using the template in VS.
The <projectname>.Mobile.csproj contains a typo that makes building impossible:

```
<Target Name="ValidateOverrides" BeforeTargets="Restore;_CheckForUnsupportedTargetFramework" Condition="'$(OverrideTargetFramework)' != ''">
    <Error
        Text="OverrideTargetFramework set to '$(OverrideTargetFramework)' is missing valid target. Set OverrideTargetFramework to one of the TargetFrameworks for this project or skip building this project (eg unload the project in Visual Studio)"
        Condition="$(OverrideTargetFramework.Contains('windows10')) or !$(OverrideTargetFrameork.Contains('-'))" />
  </Target>
```

The second "OverrideTargetFrameork" in the condition should also be "OverrideTargetFramework".

[UnoApp_Net8.Mobile.csproj.txt](https://github.com/unoplatform/uno/files/14302425/UnoApp_Net8.Mobile.csproj.txt)

**Comment by dansiegel** [comment, marked as answer]
Thanks for reporting this... this will be fixed by unoplatform/uno.templates#569
